core/arch/arm/tee/entry_std.c

```diff
@@ -64,7 +64,10 @@ static TEE_Result set_tmem_param(const struct optee_msg_param_tmem *tmem,
 	size_t sz = READ_ONCE(tmem->size);
 
 	/* Handle NULL memory reference */
-	if (!pa && !sz) {
+	/* NOTE: sz==0 is a valid case, must pass valid ptr to TA   */
```

Comment:
```
/*
 * Please format multiline
 * comments like this.
 */
```
I don't have access to GPTS case de-14-33 so I can check the claim. I'd rather see a reference to the specification as also the GP tests are based on the specifications.